src/config.py

```diff
@@ -3,13 +3,20 @@
 from pydantic import PostgresDsn, RedisDsn, model_validator
 from pydantic_settings import BaseSettings
 
-from .constants import Environment
+from src.constants import Environment
 
 
 class Config(BaseSettings):
     DATABASE_URL: PostgresDsn
     REDIS_URL: RedisDsn
 
+    JWT_ALG: str
+    JWT_EXP: int
+    JWT_SECRET: str
+
+    JWKS_CACHE: str = ""
```

Comment:
I think you can make this of type `jwk.JWKSet` that way in `auth.py` you don't have to assume it is just one key